joint2_controller:
    controller:
        package: dynamixel_controllers
        module: joint_torque_controller
        type: JointTorqueController
    joint_name: joint2
    joint_speed: 0.5
    motor:
        id: 14
        init: 2047
        min: 1011
        max: 3083
